bluesky/plan_stubs.py

```diff
@@ -342,6 +342,15 @@ def rd(obj, *, default_value=0):
         The "single" value of the device
 
     """
+    # Location is canonical if it exists
+    if isinstance(obj, Locatable):
+        location = yield Msg("locate", obj)
+        if location is None:
+            # list-ify mode
+            return default_value
+        else:
+            return location["readback"]
```

Comment:
Should `rd` return `setpoint` or `readback`?

Reply:
+1 for `readback`

Reply:
Agreed